src/sentry/models/project.py

```diff
@@ -349,6 +350,9 @@ def transfer_to(self, organization):
                 environment_id=Environment.get_or_create(self, environment_names[environment_id]).id
             )
 
+        # Manually move over organization id's for Monitors
+        Monitor.objects.filter(organization_id=old_org_id).update(organization_id=organization.id)
```

Comment:
i had error handling up for this but not totally sure what to do. should i just log it? change the slug?

Reply:
this is in case of a unique org_id + slug collision

Reply:
That's tricky. I feel like you should generate a new slug, but obviously that'll break checkins unless they update their code. I'm not sure how much better we can do than that though. If we could email them it'd be helpful, but not sure how much work we want to put into this.

Reply:
well once we update the org doesn't that change the DSN/checkin endpoint?

Reply:
yeah it does, that's a good point.

Reply:
It does, so their monitor breaks anyway. Would be still good if we had some way to tell them their slug changed. Otherwise, maybe they just update the DSN and start checking in to the wrong monitor. Probably not high priority though.

Reply:
i'm just going to put an `IntegrityError` handler and no-op with a comment